GameServer/Logic/GameObjects/Announce.cs

```diff
@@ -0,0 +1,49 @@
+﻿using LeagueSandbox.GameServer.Logic.Packets;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LeagueSandbox.GameServer.Logic.GameObjects
+{
+    public class Announce
+    {
+        private bool Announced;
+        private long EventTime;
+        private Announces MessageId;
+        private bool IsMapSpecific;
+
+        public Announce(long eventTime, Announces id, bool isMapSpecific)
+        {
+            Announced = false;
+            EventTime = eventTime;
+            MessageId = id;
+            IsMapSpecific = isMapSpecific;
+        }
+
+        public bool IsAnnounced()
```

Comment:
Not to mention the private variable naming convention is wrong here, should be `_someVariable`, or just use properties like you suggested